src/main/scala/com/typesafe/sbt/web/SbtWeb.scala

```diff
@@ -318,16 +298,34 @@ object SbtWeb extends AutoPlugin {
     nodeModules := nodeModuleGenerators(_.join).map(_.flatten).value
   )
 
+  def webJarsPathPrefix: Def.Initialize[Task[String]] = Def.task {
+    path(s"$WEBJARS_PATH_PREFIX/${moduleName.value}/${version.value}/")
+  }
+
+  def syncExportedAssets(): Def.Initialize[Task[File]] = Def.task {
+    val syncTargetDir = classDirectory.value
+    val syncRequired = TrackLevel.intersection(trackInternalDependencies.value, exportToInternal.value) match {
```

Comment:
No, this method should be renamed to something like `trackedAssets(track: TrackLevel)`, and pass in different track level from `exportedProducts`, `exportedProductsIfMissing`, and `exportedProductsNoTracking` because those are the levels of the depender. Then you can take the intersection of `track` and `exportToInternal.value`.